src/components/ThumbnailImage.js

```diff
@@ -33,73 +31,71 @@ const defaultProps = {
     imageHeight: 200,
 };
 
-class ThumbnailImage extends PureComponent {
-    constructor(props) {
-        super(props);
+/**
+ * Compute the thumbnails width and height given original image dimensions.
+ *
+ * @param {Number} width - Width of the original image.
+ * @param {Number} height - Height of the original image.
+ * @returns {Object} - Object containing thumbnails width and height.
+ */
 
-        this.updateImageSize = this.updateImageSize.bind(this);
-        const {thumbnailWidth, thumbnailHeight} = this.calculateThumbnailImageSize(props.imageWidth, props.imageHeight);
-        this.state = {
-            thumbnailWidth,
-            thumbnailHeight,
-        };
+function calculateThumbnailImageSize(width, height, windowHeight) {
+    if (!width || !height) {
+        return {thumbnailWidth: 200, thumbnailHeight: 200};
```

Comment:
Hmm, why we're returning this object now, instead of empty object previously?

Reply:
https://github.com/Expensify/App/pull/20892#discussion_r1262114113

Reply:
Thanks! I commented there https://github.com/Expensify/App/pull/20892#discussion_r1274902832